helper/staking/staking.go

```diff
@@ -10,6 +12,17 @@ import (
 	"github.com/0xPolygon/polygon-edge/types"
 )
 
+var (
+	MinValidatorCount = uint32(1)
+	MaxValidatorCount = uint32(math.MaxUint32)
+)
+
+var (
+	errInvalidValidatorNumber = errors.New("minimum number of validators can not be greater than" +
```

Comment:
Typo here, should be `than the...`

Reply:
fixed a91c866e0f6d742a0cc4af63f07196b4703ec96a